package.json

```diff
@@ -18,8 +18,9 @@
     "vscode": "^1.43.0"
   },
   "dependencies": {
-    "@zowe/cli": "6.31.1",
-    "vscode-nls": "4.1.2"
+    "@zowe/cli": "6.33.3",
+    "vscode-nls": "4.1.2",
+    "node-loader": "^2.0.0"
```

Comment:
With `cpu-features` excluded, are the `node-loader` pieces still needed here and in the webpack configs?